### DISCUSSION TITLE
The CMB and the SZ effect

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#237] [Astrophysics and Physics]
_Background:_ A nonexpanding universe should have an alternative explanation of the cosmic microwave background. Relatively recent evidence has pointed at a [correlation between the CMB and the solar system](https://en.wikipedia.org/wiki/Axis_of_evil_(cosmology)) and a [much higher temperature for the heliosheath](https://ui.adsabs.harvard.edu/abs/2019NatAs...3.1019R/abstract) than previously thought. Is it possible the CMB is actually coming from the edge of the solar system?

One problem with this is that the [SZ effect](https://en.wikipedia.org/wiki/Sunyaev%E2%80%93Zeldovich_effect) suggests it can't be. It comes from CMB photons getting a boost in energy from galaxy clusters. This is predicted, observed, and measured, and confirms the CMB cannot be local.

Here's what it looks like:

![image](https://github.com/a-cosmology-group/acg/assets/19311107/8533b35b-c3cb-4605-8755-76c252271d4e)

It's cooler in the low frequencies, and hotter in the higher frequencies. This is Abell 2319, the "poster child" for the SZ effect, as in you see this particular galaxy cluster in most places the effect is described. 

I downloaded the PlanckSZ2 database, as well as 6 full sky Planck maps at different frequencies. For each object in the SZ database I use the coordinates to make a CMB plot. After looking at about 400 (of 1653 of them in total) it seems to me very few resemble Abell 2319 at all. 

Here's the notebook, which has URLs to the data files used:

https://github.com/mikehelland/hubbles-law/blob/master/other/python/cmbsquares.ipynb

And just look at the SZ objects 400 thru 424. These are allegedly all SZ effects. But they don't look much like the poster. 

I'm not trying to go full-on crank, but could this SZ effect be mostly random noise? It's compelling in concept, but the evidence is a bit underwhelming. Thoughts?

**Comment by LifeIsStrange** [comment]
Your opensource notebooks are amazing as usual.
Note that my comment is written from my phone so formatting will be bad.

The question of whether the CMB is generated nearby or far away is primordial and I believe it can be examined from many angles. While the scientific research has made many key observations regarding the CMB in the last few years, sadly my lack of education in spectroscopy prevents me from making inferences and it seems nobody has connected all the dots.

Wiki says
> Analysis of CMB data at higher angular resolution (high 
L value) requires taking into account the Sunyaev–Zeldovich effect
Implicitly meaning that the SZ effect increase (linearly?) with the L value.

In the concise review of CMB anomalies, section III.2.9
Speaks of the high low L tension, 
> pointed out that there are internal
inconsistencies in the Planck TT power spectrum. The
ΛCDM parameter values derived by the high l part of the
CMB anisotropy spectrum (l > 1000) are in 2 − 3σ ten￾sion with the corresponding values of these parameters
derived from the low l part of the spectrum (1 < 1000).


So the funny thing is while there is so much human resources spent on the non early universe Hubble tension, the CMB is actually internally inconsistent and in tension with itself. 
For example:

> that the value of H0
predicted by Planck from l > 1000, H0 = 64.1 ± 1.7 km
s
−1M pc−1
, disagrees with the value predicted by Planck
from l < 1000, H0 = 69.7 ± 1.7 km s−1M pc−1 at the
2.3σ level. Thus it is found that the value of H0 depends
on the CMB l-range examined.

From https://arxiv.org/abs/2105.05208
While it seems intuitively the SZ effect might partially account for this new hubble tension, since the effect is well known I believe the effect is already "corrected" and that the tension is residual? Even so it might be the correction is only partial so the extent of the SZ effect to explain what is possibly the biggest CMB anomaly is an important underlooked open question it seems.


> Is it possible the CMB is actually coming from the edge of the solar system?
One problem with this is that the [SZ effect](https://en.wikipedia.org/wiki/Sunyaev%E2%80%93Zeldovich_effect) suggests it can't be. It comes from CMB photons getting a boost in energy from galaxy clusters. This is predicted, observed, and measured, and confirms the CMB cannot be local.

I don't think it is a question whether there are distant microwave sources. 
There are many distant and distinct sources that emits microwaves, the most direct one being radio galaxies point sources and quasars, that have been precisely identified.
Other distant foregrounds being:
Synchrotron radiations when cosmic rays enter magnetospheres.
The far infrared background that redshift in microwaves.
Dust rethermalization.
And the free free interaction.
Here is a concise review of most (but not all) "foregrounds" (paper is not up to date) 
https://academic.oup.com/ptep/article/2014/6/06B109/1560725?login=false

So the layman question is:
If we assume the CMB is generated nearby, 
1) can our SZ observations be consistent with an SZ effect made upon non CMB parameters/frequencies sources? Aka SZ upon non Black body temperature. 
OR
2) can our SZ observations be consistent if they act upon a CMB mimicker (black body temperature) that is point source aka that only cover some pixels in the sky, e.g. a radio galaxy. Or maybe a cluster dense in radio galaxies.
If they cannot then this would rule out a nearby (only) background CMB.

Conversely, in what is IMHO the most important paper in the decade, even without taking into account dust rethermalization, a conservative model of dust opacification shows that a primordial CMB (generated only once, at Z=1100) is empirically ruled out with current observations at 3500% above the sensitivity to detect the missing opacification. 
https://arxiv.org/abs/1706.04771
A necessary consequence of the paper is that the CMB is either nearby or continuously regenerated over distance. 
Sensitivity of the anomaly should reach 350000% in the next 5 years and even work assuming there is zero cosmic dust in the IGM (only ISM).
Note about the paper being conservative:
> The
maximum distortion increases up to zmax ∼ 7. At higher z,
the CMB is not distorted, because the model is effectively
free of dust

> I consider the intergalactic opacity at visual wavelengths
of 0.01 mag h Gpc−1
that is a twice lower value than that
reported by Xie et al. (2015). The ratio of the CMB and
visual attenuation εCMB/εV of 1×10−5
is taken from Mathis
(1990) and Draine (2003). Actually, this ratio is very low
being obtained for a steep decrease of attenuation at long
wavelengths (β = 2).
Review of future CMB studies on the distortions signature 
https://arxiv.org/abs/2307.08513
synergies between distortions and anisotropies
https://arxiv.org/pdf/1910.04619

I would like to make a related but distinct argument related to the pureness/pristinity level of an hypothetical primordial CMB. 
It is a very simple argument yet I have never seen it mentioned in the literature.
If I remember correctly, the CMB has traveled 46 billion light years and has a redshift around 1100.
The average distance between galaxies is ~6 million light years (regardless of their clustering in practice which is irrelevant here?) 
https://physics.stackexchange.com/a/394105
Of course I am making a simplification here as the distance is calculated in 3D but here I am considering the level of alteration of each "pixels" in the CMB so I will assume that if we draw a line between a CMB photon starting point until it reache earth, it will have passed through 46 billion / 6 million galaxies. 
This geometrical simplification is simplist but should give a lower bound. Moreover in practice a CMB camera "pixel" is a much larger zone than the planck scale/photon size.. 
Regardless this calculation shows that a CMB photon has approximately crossed 7666 galaxies. 
This might not seems like much but it is what the calculation shows for a static universe. 
Now assuming a redshift of 1100 means that the average galaxy distance has increased by 1100X 
So we get 8400000 galaxies per photon.
This calculation is erroneous for two reasons:
1) there were no galaxies before Z=14.32 (let's believe the myth) 
Even if that were the case, the unstructured matter would have interacted with the CMB, moreover stars and cosmic dust formed before galaxies.
To simplify, we will generously assume the CMB started at Z=14.32
2) this calculate the evolution of the diameter of the observable universe, however I am unsure if the diameter or the volume should be used? 
https://www.quora.com/What-was-the-size-of-the-universe-relative-to-its-present-day-size-at-the-time-a-Galaxy-at-redshift-z-1-emitted-the-light-we-now-see/answer/Patrick-Wooldridge?ch=15&oid=28216033&share=6427c5e7&srid=uKwsH7&target_type=answer
The author give a volume of 0.7% at Z=11.1
We will also assume that the number density of galaxies independently of the volume, is constant (same destruction and creation rate, which is empirically observed at least up to Z=2) and that this constant "density" was already reached at Z=14.32. This is of course a significant simplification.
Hence it logically follows that each photon from the primordial CMB has, since Z=14.32, passed through 7666 * 14.32 = 110000 galaxies.
And approximately ten time more using volume (but I don't think volume should be used? Sorry I'm bad at geometry).
Note that I am pretty sure the paper I linked modelled this in a much more rigorous way than I did but still, this value, the number of galaxies traveled per CMB photon is fundamental to test the viability of detecting a primordial CMB.
The question that needs to be answered is what is the effect of a single average galaxy on the CMB spectra and multiply this effect 110000 times. And see what percentage of the signal to noise ratio might remain.
And I have difficulties answering this basic part of the equation.. 
It is well known CMB pixels have temperature micro fluctuations 
> the [root mean square](https://en.m.wikipedia.org/wiki/Root_mean_square) variations are only 18 μK

On average ~18uK albeit peaks at ~100uK IIRC. 
The axis of evil is the alignment of the CMB quadrupole and octopole with the solar axis.
I don't know how to translate that into the metric of uK temperature fluctuations so I can't compute a result... 
But I suppose it could be done or that we could know that a radio galaxy is associated with a fluctuation Y. 

If we were to assume the Milky Way is representative of the average galaxy and that it induce (arbitrary number) a CMB fluctuation per pixel, of 1uK AKA 1millionth Kelvin, then it follows that each CMB photons since Z=14.32 should have an alteration of the primordial signal of 110000uK or 0.11 kelvins.
This is assuming the axis of evil induce an additive alteration of 1uK (arbitrary unsourced number) and that the axis of evil is mediated via our galaxy. If the axis of evil is mediated by the Solar system and the solar system is a representative stellar system, then doing simplistic assumptions that galaxies are like the Milky Way and that systems are single stars (not binaries) then there should be 100billion axis of evils per galaxies making the Primordial CMB signal 100% altered by multiple order of magnitudes.

So basically, intuitively the idea that we could observe a primordial CMB is laughable given how much altered the signal should be.it might be (no idea) the reason why expandists were all wrong at predicting the CMB temperature (e.g. Gamow at ~20kelvins IIRC) 
However my calculation has simplifications but most importantly the Kelvin fluctuation level per galaxy being unknown to me I cannot compute it.
If we assume the quadrupole means there are multiple ~18uK fluctuations that are cross correlated then it means the primordial CMB is ruled out by the axis of evil? 
Moreover why do we observe only one axis of evil and not as many as there are stars in the Milky Way?
Also what about the SZ effect per galaxy? 
Note that the dipole seems to be a "fluctuation" of 3.3 millikelvins which is huge but I'm not sure if I interpret this quote correctly and on a per pixel basis.

> The CMB dipole represents the largest anisotropy with the amplitude of CMB dipole (3.3621 ± 0.001 mK), e.g., [[69](https://www.ej-physics.org/index.php/ejphysics/article/view/311#ref-69)]–[[74](https://www.ej-physics.org/index.php/ejphysics/article/view/311#ref-74)]

Regardless, back on the topic on how to assess whether the CMB is nearby or not, 
A key unique probe that only very few phenomena can explain, is the CMB Birefringence, hence it has a uniquely high constraining power.
https://github.com/orgs/a-cosmology-group/discussions/231
https://arxiv.org/abs/2202.13919

I'd also like to say that dipole anisotropies are found in many probes, including quasars, which is highly intriguing.
There is also a worldwide galaxy spin rotation alignment bias.

Btw here is one of the few recent papers on a solar CMB.
https://www.ej-physics.org/index.php/ejphysics/article/view/311
That has an absolutely unique insight from Newton Cooling law (see figure 3 data)
The model in the paper has intriguing empirical backing (figure 3) and has two ways to test it, one a detectable effect via solar flares and two the peak of the power spectrum would be different based on distance (e.g. We should send a planck like satellite around jupiter, quite remarkable it has never been done even with a low resolution instrument).
I'm not sure whether having a different position for the peak of the power spectrum (but still a 2.7k temperature?) is a theory that require finetuning/coincidence or not.

**Reply by RedshiftDrift** [reply]
> If I remember correctly, the CMB has traveled 46 billion light years and has a redshift around 1100.

No, the Big Bang / $\Lambda\mbox{CDM}$ argument is that the CMB has travelled 13.8 Gyr - 380000 years, which gives it a redshift of ~1100.  The temperature then was 3000 K, now it is 3000 K/1100 = 2.7 K.

> there were no galaxies before Z=14.32

JWST has already identified [galaxies at z = 22](http://arxiv.org/abs/2311.15121) and beyond (without spectroscopic cofirmation yet, but...), why limit your estimate at z = 14.32?

About this paper:
> The [Stefan-Boltzmann Law and the Newton’s Cooling Law](https://www.ej-physics.org/index.php/ejphysics/article/view/311)

The author writes: "The Stefan-Boltzmann law then gives a temperature for the blackbody surface at the distance... $T(t)$ is the blackbody surface temperature at time t, $T_{ENV} =$ 2.5887 K (the interstellar temperature to fit the value T = 2.7255 K at R = 140 AU), R is the distance from the Earth..."

Problem is, apart for a few asteroids, there is ***no*** blackbody surface at 140 AU, so that argument does not hold water.  The loophole is that the author "proposes the existence of the Blackbody Sphere at the distance R = 140 AU" just to get the right answer, but such a black sphere obviously doesn't exist, otherwise we couldn't see anything beyond 140 AU!

**Reply by mikehelland** [reply]
I think the reasoning might be something like there is the hydrogen wall around our solar system. This absorbs energy from cosmic rays from the out side, and solar winds from the inside. It doesn't necessarily have to be a black body, in the traditional sense, it just has to radiate excess heat. 

According to [Kirchoff's law of thermal radiation](https://en.wikipedia.org/wiki/Kirchhoff%27s_law_of_thermal_radiation) absorption equals emissivity for an object in equilibrium. So that might be a problem. But the hydrogen wall (or heliopause, or termination shock, whatever the source) isn't going to be in equilibrium with its surroundings. I'm not sure if that opens up possibilities or not.

From a page by [Ned Wright](https://astro.ucla.edu/~wright/CMB.html):

![image](https://github.com/a-cosmology-group/acg/assets/19311107/baea3739-82c0-4659-9d01-9a24db7fd7b0)
 
The residuals at the bottom show there is some wiggle room for a gray body. I'm not sure if that's enough wiggle room to make something fit, but it seems to me the TS/heliopause/hydrogen wall should have some kind of detectable signature.

**Comment by mikehelland** [comment]
Re; @LifeIsStrange 

>https://arxiv.org/abs/1706.04771
>A necessary consequence of the paper is that the CMB is either nearby or continuously regenerated over distance.

This is very interesting. Thanks!

**Reply by LifeIsStrange** [reply]
The largest thermal SZ effect cluster survey released to date (today):
> measurement for the z>1 cluster subsample yields a significance of 4.3\sigma, which is the strongest CMB lensing detection in a z>1 cluster sample to date.
And 16 sigma detection for more nearby clusters.
https://arxiv.org/abs/2406.14754

Note that when the missing dust signature paper says the cmb must be either nearby or partially regenerated, by nearby it is meant closer than Z=7. (and will get necessarily closer than ~Z=3 in the next few years) or closer than Z=7 divided by 35 if we include the IGM and not just the ISM.

Again I don't think the thermal SZ effect detected at Z slightly above 1 refute the viability of a sun driven CMB. As we might assume those thermal SZ détections are non pure black body microwaves sources that are truly distant and that superimpose (foreground) on the solar system background.
But my assumption might be ruled out by someone that has expertise in spectroscopy and the SZ effect, it would be very interesting to know

**Reply by LifeIsStrange** [reply]
So on the topic of a nearby CMB one has to wonder what could be the possible sources, one being the sun heliosphere.
About another candidate:
There is a "groundbreaking" wide radio sky survey called LOFAR DR2,
a new Farraday analysis of it says:
> reveals an ordered structure across two-thirds of the observed area, whose orientation aligns well with that of both the HI filaments and the magnetic field. We estimate the minimum distance to the Faraday structures to be between 40 and 80 pc, which puts them in the vicinity of the Local Bubble wall. The emission is organised in a large gradient in Faraday depth whose origin we associate with the curved wall of the Local Bubble. Comparing our data with a model of the Local Bubble wall, we conclude that we might be probing a contribution of the medium inside the Local Bubble cavity as well, corresponding to the complex of local interstellar clouds.

So it seems this study finds a very mysterious structure that surrounds us.
https://arxiv.org/abs/2406.14679

Here is an older result about the local bubble (not to be confused with the local void)
> the 353 GHz polarized sky as observed by Planck is dominated by a close-by magnetized structure that extends between 200 and 300 pc and coincides with the shell of the Local Bubble.

https://www.aanda.org/articles/aa/abs/2019/11/aa36547-19/aa36547-19.html
So are those empirical evidence for a local bubble generated CMB?

> The Sun is embedded in the so-called Local Bubble (LB) – a cavity of hot plasma created by supernova explosions and surrounded by a shell of cold, dusty gas.

Of interest would also be studies on the most nearby superbubble (albeit it is not enclosing us) as it proves the existence of Large-scale synchrotron loops
https://www.aanda.org/articles/aa/full_html/2023/09/aa47283-23/aa47283-23.html

there is also the intriguing topic of Stellar wind bubbles as Galactic cosmic-ray re-accelerators
https://arxiv.org/abs/2403.18484

edit:
there is also the https://en.wikipedia.org/wiki/Local_Interstellar_Cloud
In 2009, [Voyager 2](https://en.wikipedia.org/wiki/Voyager_2) data suggested that the magnetic strength of the local interstellar medium was much stronger than expected (370 to 550 [pico](https://en.wikipedia.org/wiki/Pico-)[teslas](https://en.wikipedia.org/wiki/Tesla_(unit)) (pT), against previous estimates of 180 to 250 pT).
This interaction with the [heliosphere](https://en.wikipedia.org/wiki/Heliosphere) is under study by the [Interstellar Boundary Explorer](https://en.wikipedia.org/wiki/Interstellar_Boundary_Explorer) (IBEX), a [NASA](https://en.wikipedia.org/wiki/NASA) satellite mapping the boundary between the Solar System and interstellar space.

before I forget about it:
In order to have insights about a CMB generated in the sun heliosphere, there are many original and little known results from the SOHO and Cluster II satellite
https://en.wikipedia.org/wiki/Cluster_II_(spacecraft)#Selected_publications
Significant advances on the sun heliosphere signature should be possible with this little known satellite to be launched in 2025
https://en.wikipedia.org/wiki/Polarimeter_to_Unify_the_Corona_and_Heliosphere

**Reply by ExpEarth** [reply]
@LifeIsStrange 

`Note that when the missing dust signature paper says the cmb must be either nearby or partially regenerated, by nearby it is meant closer than Z=7.`

`So on the topic of a nearby CMB one has to wonder what could be the possible sources, one being the sun heliosphere.`

On one hand Vavrycuk uses Big Bang assumptions to derive his numbers.  This unfortunately makes it hard for us to take his numbers at face value.  But at the very end he says the Big Bang explanation of the CMB could be wrong and that dust absorption and reemission could explain the lack of an absorption signal in the CMB.  Since the dust would be regenerating the CMB, it could appear the CMB is coming from local sources rather than a Big Bang.  From there you can get to static model.

I would also like to point out that my shell mechanism for CMB evolution also has the property of appearing to come from local sources.  This is because CMB waves as they move from the shell transfer energy to spacetime.  In this way the CMB can retain its blackbody character, as the photon _number_  is also changing.  See #204

**Reply by mikehelland** [reply]
If you're going to show that the standard model's predictions don't match observations, you're gonna have to calculate the what the standard model predicts. That seems like the right (and only) way to make the argument.

**Reply by ExpEarth** [reply]
I think you're right and I edited my comment.

**Comment by mikehelland** [comment]
Here are 160 SZ candidates from Planck2SZ which is ~10% of the database, plus Abell 2319 for reference (index 321 in the same database):

https://mikehelland.github.io/hubbles-law/img/cmbsz/index.htm

Visually, there doesn't seem to be any obvious SZ effects.

My next step is to look at this algorithmically, because visual identification doesn't seem to be effective.

**Comment by LifeIsStrange** [comment]
Recent evidence for the CMB being local

> Recent observations have indicated a Cosmic Microwave Background (CMB) temperature decrement in the direction of local galaxies within the 2MASS Redshift Survey. We investigate this detection by analyzing its frequency dependence and sensitivity to component separation methods, suggesting that Galactic foregrounds are unlikely to be the cause. Contrary to previous studies, we find that the decrement is independent of galaxy type, indicating a possible correlation between the CMB and the overall matter density field.
https://arxiv.org/abs/2407.17599

**Comment by ExpEarth** [comment]
It seems that the temperature anomalies are really tiny, in micro Kelvins.  I don't think this supports the local CMB idea too strongly.

@LifeIsStrange 
In other posts I have suggested that, if there is a centre to the universe, and we are not quite at it but close to in the Milky Way, then there should be regions closer to the centre than we are.  Those regions could have temperatures less than 2.7 K.  Would you say they would know of such regions by now, if they existed, or is this something still possibly out there to detect?

**Reply by LifeIsStrange** [reply]
> if there is a centre to the universe, and we are not quite at it but close to in the Milky Way

reminded me of
>  it is then derived the Hubble constant dependency on the distance from the Universe’s center, which is relatively near the Milky Way galaxy.

https://www.ccsenet.org/journal/index.php/apr/article/view/47566

> Those regions could have temperatures less than 2.7 K. Would you say they would know of such regions by now, if they existed, or is this something still possibly out there to detect?

The CMB temperature has only been measured beyond the milky way, in a few places, as it requires rare high precision spectrographs and it is underfunded.
All measurements until now though shows an increase in temp with redshift, at the speed predicted by LCDM
I consider therefore this prospect as mostly ruled out unless you find a systematic in physics/light that would bias our remote temperature measurements.

The only evidence I know for a decrement is still this https://github.com/orgs/a-cosmology-group/discussions/237#discussioncomment-10159225

**Comment by LifeIsStrange** [comment]
As a followup to the missing dust signature in the CMB paper, here is a state of the art analysis of dust attenuation in high Z (7)
https://arxiv.org/abs/2501.10541

> our results show little evolution in dust properties in the REBELS sources compared to the local Universe. Comparing our recovered trends to empirical models suggests that the most important factor driving the variation in the attenuation curves in our sample is the dust-star geometry, not the properties of the dust grains themselves.
